src/lib/menu/menu-trigger.ts

```diff
@@ -313,6 +313,8 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
       const config = this._getOverlayConfig();
       this._subscribeToPositions(config.positionStrategy as ConnectedPositionStrategy);
       this._overlayRef = this._overlay.create(config);
+    } else {
```

Comment:
Flip the `if/else` here so it's something like this It should be a little easier to read through:

```
if (this._overlayRef) {
  // update position
} else {
  // create overlay
}
```